makefiles/tools/dfu.inc.mk

```diff
@@ -2,7 +2,14 @@ DFU ?= dfu-util
 FLASHER ?= $(DFU)
 HEXFILE = $(BINFILE)
 
-FFLAGS ?= -D $(HEXFILE) --reset $(DFU_ARGS)
+ifdef ROM_OFFSET
```

Comment:
`ifdef` is somehow fragile as it could be defined to an empty value and the next line would not work.